### PR TITLE
New logo

### PR DESCRIPTION
**Pull request by jurgenvinju**

### CODE REVIEW

**Comment by jurgenvinju** [comment]
It seems to work. I've done my best with the CSS and HTML tricks; not my forte (to say the least)

**Comment by jurgenvinju** [comment]
Test this by running `jekyll serve` and clicking the localhost http address that is printed on the console.

**Comment by DavyLandman** [comment]
Took me a while to run it (docker helped in the end)

But it looks good to me.

afbeelding
afbeelding

**Comment by DavyLandman** [comment]
Correction, on mobile it's not working:

afbeelding

afbeelding

File: _layouts/front.html [hunk 1]
```diff
@@ -41,10 +41,7 @@
             </div>
 
             <!-- Jumbotron -->
-            <div class="jumbotron" style="clear: left">
-               <h1>Rascal</h1>
-               <p class="lead">Metaprogramming Language</p>
-               <a class="btn btn-large btn-success" href="/start/">Get started</a>
+            <div class="jumbotron" style="clear: left; height:175px">
```
Comment (DavyLandman) [inline _layouts/front.html:44]: I think this specific height is messing with the mobile phone version.

File: assets/css/custom.css [hunk 2]
```diff
@@ -9,7 +9,9 @@
 
 
    .front-back {
-      background: no-repeat 100% 50% url('/assets/img/logo_rascal.png');
+      background: no-repeat 50% 50% url('/assets/img/logo_rascal_small.png');
+      background-position:50% 50%;  /* Sets reference point to scale from */
+      background-size: 35%;
```
Comment (DavyLandman) [inline assets/css/custom.css:14]: this is most likely messing with the mobile phone version

File: assets/css/custom.css [hunk 3]
```diff
@@ -31,7 +33,8 @@
 	text-align: center;
   }
    .front-back {
-      background: no-repeat 25% 50% url('/assets/img/logo_rascal.png');
+      background: no-repeat 25% 25% url('/assets/img/logo_rascal_small.png');
+      background-size: 100px 100px;
```
Comment (DavyLandman) [inline assets/css/custom.css:37]: explicit sizes don't work well with mobilephone I'm guessing.

**Review by DavyLandman — CHANGES_REQUESTED** [review]
It's not working on mobile.

Maybe we could use the svg instead of a png?

**Comment by jurgenvinju** [comment]
history has caught up with this pr

**Pull request closed without merging**